### PR TITLE
Use "var scope = this;" consistently throughout JS code

### LINKED ISSUE #185: Make consistent use of scoping variable name
When we assign `this` to another variable, it seems like we don't do this consistently. It would be good if we could consistently use `var scope = this` or `var self = this`.

### PR DESCRIPTION
**Pull request by fedarko**
Fixes #185.

### CODE REVIEW

File: empress/support_files/js/side-panel-handler.js [hunk 1]
```diff
@@ -256,7 +256,7 @@ define(["underscore", "Colorer"], function (_, Colorer) {
             // </p>
             // The <p> breaks lines in a way that looks nice, and the
             // label/input just define the radio buttons as is normal in HTML:
-            // see https://www.w3schools.com/tags/att_input_type_radio.asp.
+            // see https://www.w3schools.com/tags/att_input_type_radio.ascope.
```
Comment (ElDeveloper) [inline empress/support_files/js/side-panel-handler.js:259]: False positive 😄

**Comment by ElDeveloper** [comment]
Thanks @fedarko

File: empress/support_files/js/side-panel-handler.js (hunk 1 above)
Comment (fedarko) [reply empress/support_files/js/side-panel-handler.js:259]: NOOOO I can't believe this, serves me right for not being thorough enough :sweat_smile:

Let me fix that...

**Comment by fedarko** [comment]
Ok, false positive has been fixed -- should be good now :)

**Comment by kwcantrell** [comment]
Thanks @fedarko, this looks good. I think we can go ahead and merge this.

**Comment by ElDeveloper** [comment]
Thanks @kwcantrell and @fedarko!

**Pull request merged**